### PR TITLE
Setup Hello World exercise

### PR DESCRIPTION
**Pull request by pfertyk**
This is still work in progress, please don't merge. I'll add the missing pieces one-by-one, then ask for a review.

### CODE REVIEW

**Comment by iHiD** [comment]
Thanks 🙂 

Two things:
1. You can set PRs to be drafts until they're ready (see screenshot below)
2. Please get a :+1: off @ErikSchierboom before merging this first PR just to check everything looks good from an admin perspective! 🙂 

Screenshot 2023-02-26 at 22 38 58

File: .github/workflows/test.yml
Comment (BNAndras) [inline .github/workflows/test.yml:32]: General practice for Exercism now is to pin actions to a particular commit SHA.

File: exercises/practice/hello-world/hello_world_test.gd
Comment (BNAndras) [inline exercises/practice/hello-world/hello_world_test.gd:2]: Minor nitpick but the indent seems larger here than in hello_world.gd so one of them probably needs to be updated to match the other.

Comment (pfertyk) [reply exercises/practice/hello-world/hello_world_test.gd:2]: Good point, I've missed this :+1: Since Godot editor uses tabs by default, I think we should use them too (consistently, in all files). I've already changed the files :)

Side note: I think that Godot will accept both types of indentation, as long as they are consistent within a file, but there is no need to confuse GDScript learners :P

Comment (BNAndras) [reply exercises/practice/hello-world/hello_world_test.gd:2]: Okay, I think you just need to ping Erik to take a final look.

File: .github/workflows/test.yml
Comment (pfertyk) [reply .github/workflows/test.yml]: Changed to the SHA used by Crystal and Python tracks, thanks :)

File: exercises/practice/hello-world/hello_world_test.gd
Comment (BNAndras) [reply exercises/practice/hello-world/hello_world_test.gd:2]: It might be good to choose a formatter / linter if available to make sure everything is consistent between contributors and PRs. https://github.com/Scony/godot-gdscript-toolkit seems promising since it has both.  There's a GitHub Action for static analysis, but we could also use the CI for checking if a PR is well-formatted.

Comment (pfertyk) [reply exercises/practice/hello-world/hello_world_test.gd:2]: I've added a new workflow with that toolkit. It does detect some errors (e.g. an unnecessary `return`) but it doesn't detect things like mixed indent (tabs vs spaces). Nevertheless, I think it's a good idea to have the linter already configured, as it might get better in the future, and it already helps a bit. Thanks for this suggestion! ;)

**Comment by pfertyk** [comment]
@ErikSchierboom do you think this PR is ready, or should I add something?

**Comment by BNAndras** [comment]
`gradual` isn't a supported tag so the configlet CI will fail until that's either removed or configlet supports the tag being present. We shouldn't merge a PR until all the CI checks pass.

**Comment by pfertyk** [comment]
I agree :) There is a discussion in another PR about either adding `gradual` to the list of supported types, or having both `static` and `dynamic` types added to GDScript's track. If necessary, I will update the tags here after a decision is made ;)

**Comment by ErikSchierboom** [comment]
I've opened a PR to add `typing/gradual`: https://github.com/exercism/configlet/pull/842
I actually think that adding both dynamic and static typing tags makes sense too. Maybe for now just omit the gradual and update later?

**Comment by ErikSchierboom** [comment]
> @ErikSchierboom do you think this PR is ready, or should I add something?

If you (temporarily) remove the `typing/gradual` tag, I think we're good to go.

**Comment by pfertyk** [comment]
I've removed `gradual` typing and added both `static` and `dynamic` ;)

**Review by ErikSchierboom — APPROVED** [review]
Feel free to merge! Congrats on the first exercise

**Pull request merged**
